### DISCUSSION TITLE
The document of Card component disappeared.

### DISCUSSION THREAD

**Discussion by Losses** [microsoft/fluentui#16980] [Q&A] [answered]
https://developer.microsoft.com/en-us/fluentui#/controls/web/card

It told me 404...

![image](https://user-images.githubusercontent.com/1384036/107839405-9c78cc00-6de6-11eb-900a-9899cec0c907.png)

**Comment by khmakoto** [comment, marked as answer]
The `Card` component was an experimental component, as outlined in the website before. It has been deprecated and hence removed from the website. You can still access the documentation on the available props [in the repo though](https://github.com/microsoft/fluentui/blob/7.0/packages/react-cards/src/components/Card/Card.types.ts)

**Reply by Losses** [reply]
Oh, Thank you!

**Comment by jeffersoneagley** [comment]
The last time I asked about cards, the guidance was "these are basically complete and we're just waiting on a separate refactor project to complete before making them part of the main repo", now they're deprecated. What changed and where can we find fluent's card styles pre-defined?

**Reply by khmakoto** [reply]
What changed was that we are moving to a new composition and styling solution that we're basing our components off that includes improved perf and bundle size. We have no ETA on when a `Card` will be delivered under this new model unfortunately.
